### PR TITLE
first commit / landing page work

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
-  [DEMO LINK](https://nikitakarbazin.github.io/layout_miami/)

### CODE REVIEW

**Review by Kaunin1 — APPROVED** [review]
Great job! Landing adapts well for breakpoints provided in the task,  looks neat.

**Review by illiarerun — CHANGES_REQUESTED** [review]
Overall good job, but let's work on fixes a bit. Left comments in a video in your slack channel

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,459 @@
...
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#now" class="menu__link">Now on view</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contact_us" class="menu__link">Contact us</a>
```
Comment (illiarerun) [inline src/index.html:74]: still doesn't lead anywhere

**Review by illiarerun — APPROVED** [review]
Good job 🔥

**Pull request closed without merging**